debian/control

```diff
@@ -17,9 +17,11 @@ Build-Depends: debhelper (>= 9),
                libboost-test-dev,
                libboost-thread-dev,
                libboost-timer-dev,
+               libboost-random-dev,
```

Comment:
Why?

Reply:
Without it it fails to build the package on `bionic,i386` (cmake fails to find boost_random dependency). This dependency gets pulled in by state-observation. However this seems to be only used by the tests there, so a better fix would be not to export the dependency in state-observation in the first place. I'll look into it.

Reply:
I double checked, and this is used by `rigid-body-kinematics.hpp`, which is used in many places throughout `state-observation`, including in the `lipm-dcm-estimator.hpp` used by this PR. So this indeed a required dependency now.

Reply:
This dependency dates back to pre c++11, with the new random variable generators in the STL, I think we can get rid of this dependency. Do you prefer this solution?

Reply:
This is fixed in state-observation with this PR https://github.com/jrl-umi3218/state-observation/pull/7

I will merge after the tests pass

Reply:
That's better indeed. Once this is merged, we can revert c84fd31 and confirm that the pipeline passes here before merging ;)

Reply:
Now that  jrl-umi3218/state-observation#7 is merged, I've reverted the dependency change.

I'll merge when pipeline succeeds.